api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsCalculationUtils.java

```diff
@@ -24,7 +24,6 @@
 import org.joda.time.DateTime;
```

Comment:
@kaweesi please move these inline context.get...calls to the top as global variables(`evaluateWithReporting(...)` needs to be updated). Easier to see what the dependencies are for this class.